packages/rocketchat-mentions-flextab/client/views/mentionsFlexTab.js

```diff
@@ -0,0 +1,66 @@
+/*globals MentionedMessage */
+Template.mentionsFlexTab.helpers({
+	hasMessages() {
+		return MentionedMessage.find({
+			rid: this.rid
+		}, {
+			sort: {
+				ts: -1
+			}
+		}).count() > 0;
+	},
+	messages() {
+		return MentionedMessage.find({
+			rid: this.rid
+		}, {
+			sort: {
+				ts: -1
+			}
+		});
+	},
+	message() {
+		return _.extend(this, {
+			customClass: 'mentions'
+		});
+	},
+	hasMore() {
+		return Template.instance().hasMore.get();
+	}
+});
+
+Template.mentionsFlexTab.onCreated(function() {
+	this.hasMore = new ReactiveVar(true);
+	this.limit = new ReactiveVar(50);
+	return this.autorun(() => {
+		return this.subscribe('mentionedMessages', this.data.rid, this.limit.get(), function() {
+			if (MentionedMessage.find({
```

Comment:
Please, do not break que if condition, extract the find to a variable if you prefer.